src/components/TheTopNavbar.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <v-toolbar data-cy="nav-drawer" color="primary">
+    <v-toolbar-title>
+      <div class="d-flex flex-md-row flex-row-reverse align-center justify-space-between" style="cursor: pointer">
+        <TheUserMenu />
```

Comment:
Switch name to user menu.

Reply:
I discussed this with @seriouslysean and based on the [Vue style guide for singleton component instances](https://v3.ru.vuejs.org/style-guide/#single-instance-component-names-strongly-recommended) we'll stick with TheUserMenu